Sources/UIView-Shimmer/Sources/UIView+Extensions/UIView+Extensions.swift

```diff
@@ -58,11 +58,12 @@ extension UIView {
     public func setTemplateWithSubviews(_ template: Bool,
                                         color: UIColor? = nil,
                                         animate: Bool = true,
-                                        viewBackgroundColor: UIColor? = nil) {
+                                        viewBackgroundColor: UIColor? = nil, animationSpeed: CGFloat = 1.25) {
```

Comment:
could be located in new line like the others above.

Reply:
Ok, will do it. Thanks for considering my PR.